gdal/ogr/ogrsf_frmts/mitab/mitab_spatialref.cpp

```diff
@@ -260,6 +260,8 @@ const MapInfoDatumInfo asDatumInfoList[] =
 { 0,    9999,"Bosnia-Herzegovina",         10, 472.8677, 187.8769, 544.7084, -5.76198422, -5.3222842, 12.80666941, 1.54517287, 0 },
 { 6181, 9999,"Luxembourg 1930 / Gauss",     4, -192.986, 13.673, -39.309, 0.4099, 2.9332, -2.6881, 0.43, 0 },
 { 1168, 9999,"Geocentric Datum of Australia 2020", 0,-0.06155, 0.01087, 0.04019, 0.0394924, 0.0327221, 0.0328979, 0.009994,0 },
+{ 0,    1028,"Geocentric Datum of Australia 2020", 0,-0.06155, 0.01087, 0.04019, 0.0394924, 0.0327221, 0.0328979, 0.009994,0 },
```

Comment:
First column should be 1168 as well (as this is the EPSG code of the datum)
It seems to me that this second mapping is likely more idiomatic than the first one (given that it has a dedicated mapinfo datum id = 1028). And thus on the writing side, it should probably be used. Given the logic at line 1993 that selects the first match, those 2 lines should probably be inverted.

Reply:
Thanks @rouault, fixed